src/common/locale/types.js

```diff
@@ -0,0 +1,3 @@
+const LOCALE_UPDATE = 'locale_update';
```

Comment:
Note the "namespacing" of action types in the example here: https://github.com/alexnm/re-ducks

This action type should be "app/locale/UPDATE" or "app/locale/UPDATE_LOCALE"